idb/setup_postgres.sql

```diff
@@ -29,6 +29,8 @@ PRIMARY KEY ( round, intra )
 
 -- NOT a unique index because we don't guarantee txid is unique outside of its 1000 rounds.
```

Comment:
Collisions are probabilistic but SQL is absolute. If we get unlucky Indexer breaks. No need to set up that failure.

Reply:
If we have a collision, there are much larger problems to solve. Change it to unique, if we get unlucky as you say, I really need it to break.

Reply:
`algod` guarantees that txid is unique within 1000 rounds, but allows duplicates (however unlikely) outside of that. I will not make Indexer more strict such that it could fail on something `algod` will allow.

Reply:
We've had this argument before and my way isn't wrong; let it go?

Reply:
@algorotem I'm curious what error you're expecting to detect with indexer? Maybe we can handle that problem in a more direct way that doesn't risk bringing down a production service.

One thing we could do is to add that `/v2/transactions/{txid}` endpoint, and generate an error if multiple transactions are returned. We could do the same thing with account (currently that query sets a limit of 1, so we wouldn't detect it, but we can change the limit to 2).

Reply:
Brian - Feel free to not make it `UNIQUE` if it makes it easier. But, this comment cannot stay there as it is. Either remove it or modify it to what I asked for. 

Will - I don't expect any error. That's exactly the point. And if someone was able to find a collision (I invite you to try and find one) the system (not only Indexer) is, in fact, broken and there are other significant other problems to solve.

Reply:
The index must stay as it is, non-unique. What would you want out of re-wording the comment? I think it is simply true as it is. Is it suggesting or implying something we don't want? Do you have a suggested re-wording.

Reply:
possible rewrite
`-- txid is unique for 1000 rounds but may repeat after that, so index should not be unique`

Reply:
comment removed